src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -469,13 +469,33 @@ async def _schedule_single_node_session(
         log_fmt = _log_fmt.get()
         log_args = _log_args.get()
         try:
-            agent_id = scheduler.assign_agent_for_session(candidate_agents, sess_ctx)
-            if agent_id is None:
-                raise InstanceNotAvailable
+            # OP.1415 if configured sess_ctx.agent_id is existed, skip assin_agent_for_session
```

Comment:
```suggestion
            # If sess_ctx.agent_id is already set for manual assignment by superadmin, skip assign_agent_for_session().
```
When you refer the OP issue number, use the format "OP#NNN", but in this case you don't have to refer explicitly, because:
 * You can utilize `git blame` and automated history checks in various IDEs
 * The issue is not a very complicated one and does not require sophisticated/long background context to describe other than the code logic itself.

Also please use the correct English grammar and check typo.